### PR TITLE
chore: update receive ui

### PR DESCRIPTION
**Pull request by duanyytop**

### CODE REVIEW

File: packages/neuron-ui/src/components/Receive/index.tsx [hunk 1]
```diff
@@ -10,33 +10,79 @@ declare global {
...
+
+const QRCodeModal = styled.div`
   text-align: center;
 `
 
-const Receive = () => {
+const Receive = ({ address }: { address: string }) => {
   const [t] = useTranslation()
-  const address: string = '0x0da2fe99fe549e082d4ed483c2e968a89ea8d11aabf5d79e5cbf06522de6e674'
+  const [show, setShow] = useState(false)
```
Comment (ashchan) [inline packages/neuron-ui/src/components/Receive/index.tsx]: `show` is too generic. Perhaps something like `showLargeQR`?

File: packages/neuron-ui/src/components/Receive/index.tsx [hunk 2]
```diff
@@ -10,33 +10,79 @@ declare global {
...
         </Card.Header>
         <Card.Body>
-          <QRCodePanel>
-            <QRCode value={address} size={256} />
+          <AddressTip>{t('Address')}</AddressTip>
+          <AddressPanel>
+            <Address onClick={() => window.clipboard.writeText(accountAddress)}>{accountAddress}</Address>
+            <OverlayTrigger placement="bottom" overlay={<Tooltip id="address-tooltip">{t('Copy address')}</Tooltip>}>
+              <CopyImage
+                src="./image/copy.png"
```
Comment (ashchan) [inline packages/neuron-ui/src/components/Receive/index.tsx]: Have we confirmed that referring resources relatively works perfectly after packaging the app?

Comment (duanyytop) [reply packages/neuron-ui/src/components/Receive/index.tsx]: I will go to confirm now.

File: packages/neuron-ui/src/components/Receive/index.tsx (hunk 1 above)
Comment (duanyytop) [reply packages/neuron-ui/src/components/Receive/index.tsx]: OK

File: packages/neuron-ui/src/components/Receive/index.tsx (hunk 2 above)
Comment (duanyytop) [reply packages/neuron-ui/src/components/Receive/index.tsx]: Indeed, it could not work and I import `grommet-icons` to provide copy image

Comment (ashchan) [reply packages/neuron-ui/src/components/Receive/index.tsx]: Great. But we still need to figure out how to make image and other resource loading work.

Comment (duanyytop) [reply packages/neuron-ui/src/components/Receive/index.tsx]: OK. I will research this question.

File: packages/neuron-ui/src/components/Receive/index.tsx [hunk 3]
```diff
@@ -51,21 +41,30 @@ const Receive = ({ address }: { address: string }) => {
...
-          <AddressPanel>
-            <Address onClick={() => window.clipboard.writeText(accountAddress)}>{accountAddress}</Address>
-            <OverlayTrigger placement="bottom" overlay={<Tooltip id="address-tooltip">{t('Copy address')}</Tooltip>}>
-              <CopyImage onClick={() => window.clipboard.writeText(accountAddress)} />
-            </OverlayTrigger>
-          </AddressPanel>
+          <Form.Group>
+            <Form.Label>{t('Address')}</Form.Label>
+            <AddressPanel>
+              <OverlayTrigger placement="bottom" overlay={<Tooltip id="address-tooltip">{t('Copy address')}</Tooltip>}>
```
Comment (ashchan) [inline packages/neuron-ui/src/components/Receive/index.tsx]: Please make sure we're using namespace i18n keys. (Can be in a future PR to make this one get merged soon.)

Comment (duanyytop) [reply packages/neuron-ui/src/components/Receive/index.tsx]: OK

Comment (duanyytop) [reply packages/neuron-ui/src/components/Receive/index.tsx]: Neuron-ui don't use namespace i18n keys now and I will create a new issue and branch to solve this question.

Comment (BaojunCZ) [reply packages/neuron-ui/src/components/Receive/index.tsx]: yes,I am doing it

File: packages/neuron-ui/src/components/Receive/index.tsx [hunk 4]
```diff
@@ -10,33 +11,73 @@ declare global {
   }
 }
 
+const AddressPanel = styled.div`
+  dispaly: flex;
+  display: -webkit-flex; /* Safari */
```
Comment (Keith-CY) [inline packages/neuron-ui/src/components/Receive/index.tsx]: unnecessary, styled-component will add this prefix

File: packages/neuron-ui/src/components/Receive/index.tsx [hunk 5]
```diff
@@ -10,33 +11,73 @@ declare global {
...
-  margin-top: 50px;
+const QRCodeModal = styled.div`
   text-align: center;
 `
 
-const Receive = () => {
+const Receive = ({ address }: { address: string }) => {
   const [t] = useTranslation()
-  const address: string = '0x0da2fe99fe549e082d4ed483c2e968a89ea8d11aabf5d79e5cbf06522de6e674'
+  const [showLargeQRCode, setShowLargeQRCode] = useState(false)
```
Comment (Keith-CY) [inline packages/neuron-ui/src/components/Receive/index.tsx]: component shouldn't have state, all state are just management in container, like MainContent

File: packages/neuron-ui/src/components/Receive/index.tsx (hunk 4 above)
Comment (duanyytop) [reply packages/neuron-ui/src/components/Receive/index.tsx]: Without `-webkit-flex`, it don't work.

File: packages/neuron-ui/src/components/Receive/index.tsx (hunk 5 above)
Comment (duanyytop) [reply packages/neuron-ui/src/components/Receive/index.tsx]: You mean all states of components(belong to main content page) should be put in `containers/MainContent`?

Comment (ashchan) [reply packages/neuron-ui/src/components/Receive/index.tsx]: From a practice perspective, I think it's fairly fine to have simple 'state' like this within a pure component. This is only a toggle value to determine if the QR should be displayed in a large.

File: packages/neuron-ui/src/components/Receive/index.tsx (hunk 4 above)
Comment (Keith-CY) [reply packages/neuron-ui/src/components/Receive/index.tsx]: Interesting, the styled-component should add that automatically.

Leave it, it's running only in chrome, prefix is unnecessary.

File: packages/neuron-ui/src/components/Receive/index.tsx (hunk 5 above)
Comment (Keith-CY) [reply packages/neuron-ui/src/components/Receive/index.tsx]: A component with its inner state should be regarded as a widget.

There are three ways to display the larger one.

1. place the larger one in Dialog Widget, and set the dialog state in MainContent to toggle it.
2. add a new widget which has its own state to toggle the size of its picture.
3. use pure CSS to display/hide the image.

It's ok for now. Refactor it later.

**Comment by ashchan** [comment]
@duanyytop Looks good now. Could you please resolve the conflicts?

**Comment by duanyytop** [comment]
OK. I will fix conflict now.

**Pull request merged**